e2e/commands/import.e2e.js

```diff
@@ -1448,8 +1448,7 @@ describe('bit import', function () {
     });
   });
 
-  // @todo: fix this case. currently it does change it in the bit.map, but the link on the filesystem still direct to the .dependency dir
-  describe.skip('import component is-type as a dependency of is-string and then import is-type directly', () => {
+  describe('import component is-type as a dependency of is-string and then import is-type directly', () => {
```

Comment:
We should add the opposite case:
import is-type then import is-string.
while we will import is-string we will have is-type in the package.json of course, so we will install it as a package. we need to make sure that after this install we link it to the imported is-type.

Reply:
Correct. We have another task for this case: https://github.com/teambit/bit/issues/521
I guess I'll add the e2e test while fixing it.